GenderPayGap.WebUI/Controllers/Report/ReportStartingController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using GenderPayGap.Core;
+using GenderPayGap.Core.Classes;
+using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Database;
+using GenderPayGap.Database.Models;
+using GenderPayGap.WebUI.Helpers;
+using GenderPayGap.WebUI.Models.Report;
+using GenderPayGap.WebUI.Models.ReportStarting;
+using GenderPayGap.WebUI.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GenderPayGap.WebUI.Controllers.Report
+{
+    [Authorize(Roles = LoginRoles.GpgEmployer)]
+    [Route("account/organisations")]
+    public class ReportStartingController: Controller
+    {
+        
+        private readonly IDataRepository dataRepository;
+        private readonly DraftReturnService draftReturnService;
+
+        public ReportStartingController(
+            IDataRepository dataRepository,
+            DraftReturnService draftReturnService)
+        {
+            this.dataRepository = dataRepository;
+            this.draftReturnService = draftReturnService;

```

Comment:
Now that the call to this service has gone, I think it can be removed entirely from the class.